pyvista/plotting/renderer.py

```diff
@@ -695,6 +695,24 @@ def has_charts(self):  # numpydoc ignore=RT01
         """Return whether this renderer has charts."""
         return self._charts is not None and len(self._charts) > 0
 
+    def get_charts(self):  # numpydoc ignore=RT01
+        """Return a list of all charts in this renderer.
+
+        Examples
+        --------
+        .. pyvista-plot::
+           :force_static:
+
+           >>> import pyvista as pv
+           >>> chart = pv.Chart2D()
+           >>> chart.line([1, 2, 3], [0, 1, 0])
+           >>> pl = pv.Plotter()
+           >>> pl.add_chart(chart)
+           >>> print(chart is pl.renderer.get_charts()[0])  # True
```

Comment:
We need to write output.
```suggestion
           >>> print(chart is pl.renderer.get_charts()[0])
           True
```

Reply:
I think the `print` is not necessary either